src/lib.rs

```diff
@@ -403,9 +403,9 @@ impl SimConnector {
         }
     }
 
-    pub fn request_data_on_sim_object(&self, request_id: SIMCONNECT_DATA_REQUEST_ID, define_id: SIMCONNECT_DATA_DEFINITION_ID, object_id: SIMCONNECT_OBJECT_ID, period: SIMCONNECT_CLIENT_DATA_PERIOD) -> bool {
+    pub fn request_data_on_sim_object(&self, request_id: SIMCONNECT_DATA_REQUEST_ID, define_id: SIMCONNECT_DATA_DEFINITION_ID, object_id: SIMCONNECT_OBJECT_ID, period: SIMCONNECT_CLIENT_DATA_PERIOD, flags: SIMCONNECT_DATA_REQUEST_FLAG) -> bool {
         unsafe {
-            let result = SimConnect_RequestDataOnSimObject(self.sim_connect_handle, request_id, define_id, object_id, period, 0, 0, 0, 0);
+            let result = SimConnect_RequestDataOnSimObject(self.sim_connect_handle, request_id, define_id, object_id, period, flags, 0, 0, 0);
```

Comment:
should probably include the other parameters as well - origin, interval, and limit. silly me for leaving out parameters that's specific for my application. if you notice missing parameters anywhere else, feel free to add those.